app/experimenter/nimbus-ui/src/gql/experiments.ts

```diff
@@ -190,3 +190,20 @@ export const GET_EXPERIMENT_QUERY = gql`
     }
   }
 `;
+
+export const GET_ALL_EXPERIMENTS = gql`
+  query GetAllExperiments {
```

Comment:
Should this return _all_ experiments? Or only the experiments for the currently LDAP logged in user?

Reply:
all!!

Reply:
In that case, do we want to also query [`owner { username }`](https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v5/graphql#query=%7B%0A%20%20%23%20experiments(limit%3A%2050%2C%20status%3A%20Accepted)%20%7B%0A%20%20experimentBySlug(slug%3A%20%22do-not-delete-about-welcome-reference-test%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20owner%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20username%0A%20%20%20%20%20%20firstName%0A%20%20%20%20%20%20lastName%0A%20%20%20%20%7D%0A%20%20%20%20status%0A%20%20%20%20name%0A%20%20%20%20slug%0A%20%20%20%20publicDescription%0A%20%20%20%20isPaused%0A%20%20%20%20proposedDuration%0A%20%20%20%20proposedEnrollment%0A%20%20%20%20populationPercent%0A%20%20%20%20totalEnrolledClients%0A%20%20%20%20firefoxMinVersion%0A%20%20%20%20application%0A%20%20%20%20channels%0A%20%20%20%20hypothesis%0A%20%20%20%20targetingConfigSlug%0A%20%20%20%20startDate%0A%20%20%20%20endDate%0A%20%20%7D%0A%7D%0A) so we can display the email address of the experiment owner (or at least the username portion of the email address) in the table?

There also seems to be a `owner { firstName, lastName }` in the GraphQL schema, but those were both empty strings on the experiments I found with a defined owner.